AppInsightsCommon/src/TelemetryItemCreator.ts

```diff
@@ -39,8 +39,8 @@ export class TelemetryItemCreator {
 
         let telemetryItem: ITelemetryItem = {
             name: envelopeName,
-            timestamp: new Date(),
-            instrumentationKey: "", // this will be set in TelemetryContext
+            time: new Date().toISOString(),
+            ikey: "", // this will be set in TelemetryContext
```

Comment:
please update to iKey. This is possibly documented wrong in the 1ds page.

Reply:
updated